tests/Feature/QuestionApiControllerTest.php

```diff
@@ -59,4 +61,25 @@ public function test_create_question_non_existing_quiz_id(): void
         $response->assertStatus(422);
         $response->assertJsonValidationErrors(['quiz_id']);
     }
+
+    public function test_delete_question_success(): void
+    {
+        $question = Question::factory()->create(['id' => 1]);
+        $response = $this->deleteJson('/api/questions/1');
+        $response->assertStatus(200)
+            ->assertJson(function (AssertableJson $json) {
+                $json->has('message')
+                    ->where('message', 'Question deleted');
+            });
```

Comment:
There's ane extra assertion you can use here to make sure the question is no longer in the database.

Works exactly the same as `$this->assertDatabaseHas()`, you can use `$this->assertDatabaseMissing()`.

i would just make sure that `$question->id` is not in the questions table anymore